### PR TITLE
Improve CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by nastra**
This change is [Reviewable](https://reviewable.io/reviews/projectnessie/nessie/1163)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -36,14 +36,14 @@ on the new feature or improvement.
...
+for more information. Small changes don't require an issue. However, it is good practice to open up an issue for
 larger changes. If you are unsure of where to start ask on the slack channel or look at [existing issues](https://github.com/projectnessie/nessie/issues).
-The 'good first issue' label marks issues that are particularly good for people new to the codebase.
+The [good first issue](https://github.com/projectnessie/nessie/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) label marks issues that are particularly good for people new to the codebase.
 
 ### Style guide
 
 Changes must adhere to the style guide and this will be verified by the continuous integration build. Java code style is approximately 
-google style and can be checked with checkstyle. Python adheres to the pep8 standard.
+[Google style](https://google.github.io/styleguide/javaguide.html) and can be checked with Checkstyle. Python adheres to the pep8 standard.
```
Comment (snazy) [inline CONTRIBUTING.md:46]: Hm, unsure whether we should link to the Google-Code-Style here, because it's "just approximately" like that.

Comment (nastra) [reply CONTRIBUTING.md:46]: While reading this paragraph I felt the immediate need to check out the Google Style Guide and I was curious how alike both style guides are (I'm fine to remove the link though). 
Also I was wondering whether there is the same Checkstyle code formatter available in order to format the code in the IDE as I'm sure other people will run into the same issue. Right now it looks like Checkstyle and the IDE formatter produce different results, which is less than ideal for day-to-day dev work.

Comment (nastra) [reply CONTRIBUTING.md]: fixed

**Comment by laurentgo** [comment]
Thanks @nastra for your contribution. May I ask you to make the title and the description more descriptive of your change? "Minor improvements" is quite generic and could apply to anything...

**Comment by nastra** [comment]
> Thanks @nastra for your contribution. May I ask you to make the title and the description more descriptive of your change? "Minor improvements" is quite generic and could apply to anything...

@laurentgo sure thing, updated PR and commit title.

**Review by snazy — COMMENTED** [review]
+1

**Pull request merged**